Subject: Handle SQL quit event
From: Ansgar Burchardt <ansgar@43-1.org>
Date: Sat Dec 13 15:00:01 2008 +0100

This patch makes Jump'n'bump quit when pressing the close button or
using other means to ask Jump'n'bump to quit (e.g. Alt+F4).

Closes: #487333

--- a/sdl/interrpt.c
+++ b/sdl/interrpt.c
@@ -419,6 +419,9 @@ int intr_sysupdate()
 				break;
 			}
 			break;
+		case SDL_QUIT:
+			deinit_program();
+			break;
 		default:
 			break;
 		}
